dashboard/v1.1/src/pages/Config/Config.tsx

```diff
@@ -42,84 +55,77 @@ const Config: FC<{}> = () => {
-        return resp.json();
-      }
-    );
-    const { data } = response;
-    let intervals: any = [];
-    data.forEach(interval => {
-      intervals.push({
-        test: interval['Test'],
-        duration: interval['Duration'],
-        unit: interval['Type']
-      });
-    });
-    setConfigIntervals(intervals);
-  };
-
-  const fetchConfigRoutes = async () => {
-    const response = await fetch(`${HOST_IP}/get-config-routes`).then(resp => {
-      return resp.json();
-    });
-    const { data } = response;
-    let configRoutes = new Map();
-    data.forEach(route => {
-      const uri = route['URL'] + '/' + route['Route'];
-      if (!configRoutes.has(uri)) {
-        configRoutes.set(uri, [route['Method']]);
-      } else {
-        configRoutes.set(uri, [...configRoutes.get(uri), route['Method']]);
-      }
-    });
-    setConfigRoutes(Array.from(configRoutes));
-  };
+  const { response, error } = useFetch<service_states>(
+    `${HOST_IP}/service-state`
+  );
 
   useEffect(() => {
-    fetchConfigIntervals().then(() => fetchConfigRoutes());
+    fetchConfigIntervals(setConfigIntervals).then(() =>
+      fetchConfigRoutes(setConfigRoutes)
+    );
   }, []);
 
-  const getTableRoutes = (routes: [string, string[]][] | null) => {
+  const getTableData = (routes: [string, routeOptionsInterface[]][]) => {
     let tableData: TableRouteType[] = [];
-    routes?.forEach(r => {
+    routes.forEach(route => {
+      let methods: string[] = [];
+      route[1].forEach(option => {
+        Object.keys(option).forEach(k => {
+          if (k === 'method') {
+            methods.push(option[k]);
+          }
+        });
+      });
       tableData.push({
-        route: r[0],
-        methods: r[1]
+        route: route[0],
+        methods
       });
     });
     return tableData;
   };
 
-  const columns = [
-    {
-      field: 'route',
-      title: 'Route'
-    },
-    {
-      field: 'methods',
-      title: 'Methods',
-      render: (rowData: { methods: any[]; route: string }) =>
-        rowData.methods.map(m => (
-          <Chip
-            key={rowData.route + m}
-            variant="outlined"
-            color="primary"
-            label={m}
-          />
-        ))
-    }
-  ];
+  const updateConfigRoutes = routes => {
+    const { data } = routes;
+    let configRoutes = new Map();
+    data.forEach((route: route) => {
+      const uri = route.URL;
+      if (!configRoutes.has(uri)) {
+        configRoutes.set(uri, [
+          {
+            method: route.Method,
+            body: route.Body,
+            headers: route.Header,
+            params: route.Params
+          }
+        ]);
+      } else {
+        configRoutes.set(uri, [
+          ...configRoutes.get(uri),
+          {
+            method: route.Method,
+            body: route.Body,
+            headers: route.Header,
+            params: route.Params
+          }
+        ]);
+      }
+    });
+    setConfigRoutes(configRoutes);
+  };
```

Comment:
Let's have these long functions to a different file in the same directory as `handles.ts`.